lib/src/vector_math/error_helpers.dart

```diff
@@ -12,8 +12,8 @@ double relativeError(dynamic calculated, dynamic correct) {
     final diff = (calculated - correct).abs().toDouble();
     return diff / correct;
   }
-  // ignore: return_of_invalid_type
-  return calculated.relativeError(correct);
+  // ignore: avoid_dynamic_calls
+  return calculated.relativeError(correct) as double;
```

Comment:
Is there a perf implication on dart2js here?

cc @rakudrama

Reply:
OH! Good point...

Reply:
Why do you need `as double`?
I would expect there is an implicit downcast from `dynamic` .

Reply:
We have a lint to eliminate implicit downcasts

Reply:
I can just revert this line. Not an issue.